tests/integration/test_collator.py

```diff
@@ -56,7 +61,7 @@ class DescribePayloadOrderCollator:
     @pytest.mark.parametrize(
         "element_ids, anchors, expected_value",
         (
-            ((9, 3, 7), ("bottom", 3, 3, "top"), (-1, 0, 1, -3, -2, 2, -4)),
+            ((1, 2, 3), ("bottom", 3, 3, "top"), (-1, 0, 1, 2, -3, -2, -4)),
```

Comment:
Can't tell if this test broke or this was just cleanup?

Reply:
cleanup cause now the display order of the payload class does not use the subtotals_with_payload_order